src/discussions/learners/learner/LearnerFilterBar.jsx

```diff
@@ -28,10 +28,10 @@ const ActionItem = React.memo(({
     style={{ cursor: 'pointer' }}
     aria-checked={value === selected}
     // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-    tabIndex={value === selected ? '0' : '-1'}
+    tabIndex={'0'}
```

Comment:
I checked the git blame and it looks like this line was added for https://github.com/openedx/frontend-app-discussions/pull/483

Can you confirm the issue that PR was fixing is still fixed with this change?

Reply:
Yes, I’ve verified that the issue addressed in PR #483 is still resolved with the current change.
Thanks @brian-smith-tcril  for pointing it out and for your thorough review!